internal/api/constraint.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+type Operator string
+
+const (
+	OperatorIn    Operator = "IN"
+	OperatorNotIn Operator = "NOT IN"
```

Comment:
Is this a typo? In the protocol you will get `NOT_IN`, see https://github.com/Unleash/client-specification/pull/5/files#diff-07b98f0d3424fd4c9547b512848b9521R67

Reply:
Wasn't really a typo since I read it in the Google Docs for the Constraints protocol you sent:
https://docs.google.com/document/d/1WxdODm1xKLaArucD7E5nHg4nruftRfkaWibaGV32GaI/edit
I'll fix mine if you fix yours 😂

Reply:
Oh this is embarrassing, fixed now.